### PR TITLE
upgrade and using delegate, rake task

### PR DESCRIPTION
**Pull request by duypn-2245**

### CODE REVIEW

**Comment by duypn-2245** [comment]
- Vẫn hiện form tạo task khi invalid 
image

**Comment by duypn-2245** [comment]
image

**Comment by duypn-2245** [comment]
ready

**Comment by duypn-2245** [comment]
https://edu-redmine.sun-asterisk.vn/issues/25932

File: app/controllers/admin/courses_controller.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ def create
   end
 
   def index
-    @courses = Course.newest.avaiable.paginate(page: params[:page])
+    @courses = current_user.courses.newest.avaiable.paginate(page: params[:page])
```
Comment (vanvtt-0952) [inline app/controllers/admin/courses_controller.rb:24]: tào lao =))

**Pull request merged**